### PR TITLE
Add warning messages for situations where the user might have forgott…

### PR DESCRIPTION
**Pull request by carolhmj**
…en to import a file.
Related forum issue: https://forum.babylonjs.com/t/weird-issue-using-vite/46468

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14635/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14635/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14635/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14635/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14635/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14635/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14635/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14635/merge#BCU1XR#0

File: packages/dev/core/src/Materials/Textures/videoTexture.ts [hunk 1]
```diff
@@ -192,7 +192,10 @@ export class VideoTexture extends Texture {
         this._currentSrc = src;
         this.name = name || this._getName(src);
         this.video = this._getVideo(src);
-        this._externalTexture = this._engine?.createExternalTexture(this.video) ?? null;
+        this._externalTexture = (this._engine?.createExternalTexture && this._engine.createExternalTexture(this.video)) ?? null;
```
Comment (Popov72) [inline packages/dev/core/src/Materials/Textures/videoTexture.ts:195]: `this._externalTexture === null` is valid, it's what you will get in WebGL. Only WebGPU uses external textures and will return a non null value.

**Review by Popov72 — CHANGES_REQUESTED** [review]

File: packages/dev/core/src/Engines/Extensions/engine.cubeTexture.ts [hunk 2]
```diff
@@ -365,6 +365,9 @@ ThinEngine.prototype.createCubeTextureBase = function (
     const extension = forcedExtension ? forcedExtension : lastDot > -1 ? rootUrlWithoutUriParams.substring(lastDot).toLowerCase() : "";
 
     let loader: Nullable<IInternalTextureLoader> = null;
+    if (ThinEngine._TextureLoaders.length === 0) {
```
Comment (sebavan) [inline packages/dev/core/src/Engines/Extensions/engine.cubeTexture.ts:368]: This is valid if you specify a list of textures as it won t use external loaders.

**Comment by sebavan** [comment]
Let me close and I ll add a way to bypass the error on create external texture in my next PR.

**Pull request closed without merging**